docs/source/software/packagemanagers/conda.md

```diff
@@ -74,6 +77,51 @@ To delete a Conda environment and all of its related packages, run:
 conda remove -n yourenvironmentname --all
 :::
 
+### Listing Environments
+You can view the environments you've created in your home directory by using the following command
+:::{code} bash
+conda env list
+:::
+
+:::{code} bash
+# conda environments:
+#
+MlGenomics               $HOME/.conda/envs/MlGenomics
+base                     $HOME/miniconda3
```

Comment:
Above we suggest using /work. Should be be consistent here or is this the intent (perhaps add note if so)